examples/tutorials/setup_poc.ipynb

```diff
@@ -104,283 +101,278 @@
-    "\n",
-    "* -i project.yml \n",
-    "\n",
-    "This optional allows you experiment the your project.yml locally, try local before used in production. \n",
-    "\n",
-    "* All specified options will resulting an project.yml file located at \n",
     "\n",
-    "<poc_workspace>/project.yml \n",
-    "\n",
-    "for example, ```/tmp/nvflare/poc/project.yml```\n",
-    "\n",
-    "\n",
-    "Let's try out some of options\n",
-    "\n",
-    "* **Named Clients**\n",
-    "\n",
-    "instead of using default  site-1, site-2 etc. user can specified the client names"
+    "This will create docker.sh for each client and server which will run `docker pull docker_image` from docker hub, then running the docker in detached mode. The docker_image must use nvflare. For example, we use the following docker image \"nvflare/nvflare\" which has nvflare preinstalled.\n",
+    "> Note: you should build your own image if you want to include your dependencies such as pytorch etc.\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "95749ed0-7e9d-4994-abe2-341fd299f801",
-   "metadata": {
-    "tags": []
-   },
+   "id": "7810bcc3",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "! echo 'y' | nvflare poc prepare -c hospital-1 hospital-2\n"
+    "! echo 'y'| nvflare poc prepare -d 'nvflare/nvflare'"
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "e4e559b4-6fff-44a2-aeb1-6b96e63888f5",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ff2755e1",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "Now, let's look at the result, if you have ```tree``` command installed in linux (``` pip install tree```), you can use it. otherwise, use ```ls -al```. "
+    "! tree /tmp/nvflare/poc/example_project/prod_00"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "7cc85def-f768-4c49-883f-8bcfd13461da",
-   "metadata": {
-    "tags": []
-   },
-   "outputs": [],
+   "cell_type": "markdown",
+   "id": "9d3a3c06",
+   "metadata": {},
    "source": [
-    "! tree /tmp/nvflare/poc/example_project/prod_00"
+    "Notice the new \"docker.sh\", now each client will using docker run with we specified \"nvflare/nvflare\" docker image"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "89add7ef-346b-40c9-97be-aa49c624adbb",
+   "id": "b6b968e9",
    "metadata": {},
    "source": [
-    "Now two clients are named \"hospital-1\" and \"hospital-2\". \n",
+    "### Homomorphic Encryption (HE)\n",
     "\n",
-    "* **Setup NVFLARE in docker mode**\n",
-    "\n",
-    "Normally, when we using ```nvflare poc start```, the clients and server are started as process in the local host. \n",
-    "What if you would like to experiment runing client and server in dockers ? You can do it now, \n",
+    "The -he option enables homomorphic encryption.\n",
     "\n",
     "```\n",
-    "nvflare poc prepare -n 2 -d  <docker_image>\n",
-    "\n",
+    "nvflare poc prepare -he\n",
```

Comment:
Changes in this PR LGTM,

I just have a general question:
As POC grows complicated, it is trying to get all the functionalities that our provision has, why don't we just ask people to do provision - deployment after simulator?

Reply:
Good question. The main thing is POC mode allows you to test all the features of a full FLARE deployment on a single machine (using the same project.yml), but with much less overhead than a real deployment.